client/checkout/blocks/confirm-upe-payment.js

```diff
@@ -45,6 +52,21 @@ export default async function confirmUPEPayment(
 			},
 		};
 
+		if ( PAYMENT_METHOD_IDS.AFTERPAY_CLEARPAY === selectedUPEPaymentType ) {
```

Comment:
We need a short comment that Afterpay differs from other PMs and expects shipping details to be supplied.